src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts

```diff
@@ -36,7 +36,13 @@ class timeSeriesModel {
         });
         
         this.timestamp.extend({
-            required: true
+            required: true,
+            validation: [
+                {
+                    validator: () => this.timestamp().utc().format() !== "Invalid date",
```

Comment:
please never relay on magic string values. What if moment add locale support or we enable that? Code won't work. 

Looks like we can use this:
![image](https://user-images.githubusercontent.com/1250431/83978576-ccdba280-a908-11ea-9b5a-9aa38e8df831.png)

Reply:
done